src/gluonts/evaluation/_base.py

```diff
@@ -31,6 +31,10 @@
 import numpy as np
 import pandas as pd
 
+from gluonts.gluonts_tqdm import tqdm
+from gluonts.model.forecast import Forecast, Quantile
+from gluonts.model.naive_2 import naive_2
```

Comment:
This requires `statsmodules` which is not installed by default

Reply:
Changed where it's imported